controllers/src/test/scala/cromwell/pipeline/controller/UserWithCredentialsControllerTest.scala

```diff
@@ -17,7 +17,7 @@ import org.scalatestplus.mockito.MockitoSugar
 
 import scala.concurrent.Future
 
-class UserControllerTest
+class UserWithCredentialsControllerTest
```

Comment:
Can we please keep `UserControllerTest` ? Same for `UserController`, `UserServiceTest` etc. As we already discussed, these classes are not supposed to work with `UserWithCredentials`, so why call them that way?

Reply:
@myazinn done